client/src/client.rs

```diff
@@ -1649,8 +1652,12 @@ mod tests {
                     .parse()
                     .expect("This account ID should be valid"),
             ),
-            torii_api_url: Some(SmallStr::from_str(DEFAULT_API_URL)),
-            torii_telemetry_url: Some(SmallStr::from_str(DEFAULT_TORII_TELEMETRY_URL)),
+            torii_api_url: Some(format!("http://{DEFAULT_API_ADDR}").parse().unwrap()),
+            torii_telemetry_url: Some(
+                format!("http://{DEFAULT_TORII_TELEMETRY_ADDR}")
+                    .parse()
+                    .unwrap(),
+            ),
```

Comment:
Not a huge fan of calling `format!(//..).parse()`.

Reply:
Me neither, just no other way to construct `url::Url`. I can look up a replacement URL implementation if you feel this is way to unseemly

Reply:
Come to think of it. Leave it for a separate ticket.

Reply:
Opened https://github.com/hyperledger/iroha/issues/3385